2020/prep/find_broken_fp.sh

```diff
@@ -9,6 +9,9 @@ if [ "$#" == 0 -o \! -d "$1" ]; then
   exit
 fi
 
-(cd $1/incremental; find *FP -name \*.smt2 | \
+# find all benchmarks contains to_fp with a decimal in an FP logic
+# (these should have been categorized as FPLRA)
+# fix the name to upper case and add ./ to beginning using perl regex.
+(cd $1/incremental; find *[fF][pP] -name \*.smt2 | \
```

Comment:
Is it an overkill to do ```find *[fF][pP]* -name \*.smt2 |grep -v '[lL][iI]\?[rR][aA]\|[nN][iI]\?[rR][aA]'```?

Reply:
You need to be careful to only grep benchmark part...

I think it's overkill. This is just something for this year to find benchmark that are in the wrong category, not something we want to maintain over the years. So lets keep it simple